Source/WebCore/platform/graphics/skia/PatternSkia.cpp

```diff
@@ -31,9 +31,9 @@
 #include "ImageBuffer.h"
 #include "NativeImage.h"
 #include <skia/core/SkImage.h>
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_BEGIN
 #include <skia/core/SkMatrix.h>
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
+WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_END
```

Comment:
Seems to me that we should move this out of the big paragraph of includes when it needs to be wrapped with macros like this. But I suppose we shouldn't mess with it if it's working.

Reply:
Ok.